apps/src/templates/certificates/CertificateBatch.jsx

```diff
@@ -21,72 +85,128 @@ export default function CertificateBatch({
   };
 
   return (
-    <div style={styles.wrapper}>
-      <h1>{i18n.printBatchCertificates()}</h1>
-      <div style={styles.imageWrapper}>
-        <img src={imageUrl} width={240} height={170} />
-        <span style={styles.instructions}>
-          <SafeMarkdown markdown={i18n.enterCertificateNames({courseTitle})} />
-          {i18n.wantBlankCertificateTemplate()}{' '}
-          <a href={imageUrl}>{i18n.printOneCertificateHere()}</a>
-        </span>
+    <div className={style.wrapper}>
+      <div className={style.headerContainer}>
+        <h1 className={style.header}>{i18n.printBatchCertificates()}</h1>
+      </div>
+      <div className={style.certificateContainer}>
+        <div className={style.imageWrapper}>
+          <img src={imageUrl} width={'100%'} />
```

Comment:
Add an `altText` field for the `<img>` please (it can be empty since the image doesn't add any new information)